packages/edit-site/src/components/editor/index.js

```diff
@@ -84,44 +86,71 @@ const siteIconVariants = {
 };
 
 export default function EditSiteEditor( { isPostsList = false } ) {
```

Comment:
My ultimate goal after all the routes get refactored is to remove this intermediary component entirely. For now, I only removed its usage from the "home" route.